requirements.txt

```diff
@@ -9,4 +9,6 @@ pre-commit
 jsonschema
 pyaudio
 Sphinx
-sphinx_rtd_theme
\ No newline at end of file
+sphinx_rtd_theme
+
+-i https://test.pypi.org/simple/ --no-deps pyodas-JacobKealey
```

Comment:
I tried doing it in my branch and I had problems with the github tests, might need to further look into that.